lib/cretonne/src/simple_gvn.rs

```diff
@@ -7,6 +7,9 @@ use ir::{InstructionData, Function, Inst, Opcode, Type};
 use scoped_hash_map::ScopedHashMap;
 use timing;
 
+#[cfg(feature = "no_std")]
+use alloc::Vec;
```

Comment:
Could this just be `use std::vec::Vec`, so that it doesn't need the `#[cfg(...)]` part?